src/plugin/area/geojsonareaimportui.js

```diff
@@ -0,0 +1,88 @@
+goog.provide('plugin.area.GeoJSONAreaImportUI');
+
+goog.require('os.query.ui.AreaOptionsStep');
+goog.require('os.ui.im.FileImportUI');
+goog.require('os.ui.window');
+goog.require('plugin.area.geojsonAreaImportDirective');
+goog.require('plugin.file.geojson.GeoJSONParserConfig');
+goog.require('plugin.file.geojson.geojsonImportDirective');
+
+
+
+/**
+ * @extends {os.ui.im.FileImportUI.<plugin.file.geojson.GeoJSONParserConfig>}
+ * @constructor
+ */
+plugin.area.GeoJSONAreaImportUI = function() {
+  plugin.area.GeoJSONAreaImportUI.base(this, 'constructor');
+
+  // file contents are only used in memory, not loaded from storage
+  this.requiresStorage = false;
+};
+goog.inherits(plugin.area.GeoJSONAreaImportUI, os.ui.im.FileImportUI);
+
+
+/**
+ * @inheritDoc
+ */
+plugin.area.GeoJSONAreaImportUI.prototype.getTitle = function() {
+  return 'Area Import - GeoJSON';
+};
+
+
+/**
+ * @inheritDoc
+ */
+plugin.area.GeoJSONAreaImportUI.prototype.launchUI = function(file, opt_config) {
+  plugin.area.GeoJSONAreaImportUI.base(this, 'launchUI', file, opt_config);
+  var steps = [
+    new os.query.ui.AreaOptionsStep()
+  ];
+
+  var config = new plugin.file.geojson.GeoJSONParserConfig();
+
+  // if a configuration was provided, merge it in
+  if (opt_config) {
+    this.mergeConfig(opt_config, config);
+  }
+
+  config['file'] = file;
+  config['title'] = file.getFileName();
+
+  try {
```

Comment:
I think this may have come from `geojsonimportui.js`. For area import, you don't need to do mapping detection. We only care about the geometry (areas discard all other metadata) and GeoJSON defines where that is by spec. I believe this whole try/catch block can be removed.

As background, mappings are for fields that are not defined by spec and need to be detected from arbitrary metadata fields. Common examples are time and ellipse semi major/minor & orientation. Ellipse fields _could_ be detected and used to create areas from point data, but that's beyond the scope of this ticket.